### PR TITLE
Feat : Add Wishlist Page

### LINKED ISSUE #6: Add Wishlist Page

### PR DESCRIPTION
**Pull request by ZuberDunge**
image

### CODE REVIEW

File: style/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: var(--primary-color-dark);
+    color: var(--white);
+    width: 2rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.wish-list-container .bat-card img {
+    width: 120px;
```
Comment (Shivanshsharma01) [inline style/wishlist.css:49]: can use rem here

**Review by GuptaEesh — APPROVED** [review]
Good use of variables and greatly written CSS.

File: style/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
+.wish-list-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 2rem;
+    max-width: 1100px;
```
Comment (Shivanshsharma01) [inline style/wishlist.css:5]: rem here too

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Can use rems above 16px, Good to merge

File: style/wishlist.css (hunk 1 above)
Comment (ZuberDunge) [reply style/wishlist.css:49]: Thanks for the review Shivansh. I'll make sure to use rem from next time.

**Pull request merged**